app/src/main/java/com/duckduckgo/app/statistics/api/PixelService.kt

```diff
@@ -30,6 +31,7 @@ interface PixelService {
         @Path("pixelName") pixelName: String,
         @Path("formFactor") formFactor: String,
         @Query(AppUrl.ParamKey.ATB) atb: String,
-        @QueryMap additionalQueryParams: Map<String, String?> = emptyMap()
+        @QueryMap additionalQueryParams: Map<String, String?> = emptyMap(),
+        @Query(AppUrl.ParamKey.DEV_MODE) devMode: Int? = if (BuildConfig.DEBUG) 1 else null
```

Comment:
Would this add the `test` param to every request e.g test=null on release builds?

Reply:
Can we move this to the `defaultParameters` section in Pixel.kt so all the default params are grouped together.

Reply:
>Would this add the test param to every request e.g test=null on release builds?

No; null values are omitted

>Can we move this to the defaultParameters section in Pixel.kt so all the default params are grouped together.

That's a nice idea, but I want to defer it. Declaring it differently to how we do it in `StatisticsService` isn't best for consistency, so if changing the way we add the header, i'd rather do it in both places. And I don't want to adopt that overhead of changing and testing `StatisticsService` right now, given the other work i'm currently focussed on.

Reply:
Looking at it again, I'm happy to forget about that suggestion entirely as other requests e.g atb.js use the pattern you have used.